app/templates/application.hbs

```diff
@@ -1,12 +1,15 @@
-<div class="pusher">
-  {{nav-bar}}
-  {{#if (eq routing.currentRouteName 'index')}}
-    {{welcome-header}}
-  {{/if}}
-  <div id="main-container" class="ui container main-container {{unless device.isInternetExplorer 'flexed-container' ''}}">
-    {{outlet}}
+{{#side-bar}}
+  <div class="footer-pusher">
```

Comment:
@niranjan94 This <div class="footer-pusher"> was moved here to ensure that side bar's pusher content will include the the original navbar with the rest of the body?